app/src/main/java/com/doubtless/doubtless/screens/answers/AnswersViewModel.kt

```diff
@@ -27,6 +27,9 @@ class AnswersViewModel(
     private val _publishAnswerStatus = MutableLiveData<PublishAnswerUseCase.Result>()
     val publishAnswerStatus: LiveData<PublishAnswerUseCase.Result> = _publishAnswerStatus
 
+    private val _publishAnswerLoading = MutableLiveData<Boolean>()
+    val publishAnswerLoading: LiveData<Boolean> = _publishAnswerLoading
```

Comment:
wrap this on Resource like the other viewmodels are doing. It will have an error success and loading state.

Reply:
also clear the livedata value when observers observe it.